src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -133,13 +132,18 @@
 results = runner.run(suite);
 failed = any([results.Failed]);
 
+function preparePath(path)
+dir = fileparts(path);
+dirExists = isempty(dir) || exist(dir,'dir') == 7;
+if ~dirExists		
+    mkdir(dir);		    
+end
 
 function tapToFile = getTapResultFile(resultsDir)
```

Comment:
This variable should be called something like tapFile.

Reply:
Done